### ISSUE TITLE
no-restricted-paths - absolute path expected where relative path is provided 

### ISSUE THREAD

**Issue by lfender6445** [benmosher/eslint-plugin-import#1890]
https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/no-restricted-paths.js#L72

```js
// returns a relative path, are we sure thats what we want? 
const absoluteImportPath = resolve(importPath, context) 
```

Here we use `resolve` from `utils/resolve` which returns a relative path

https://github.com/benmosher/eslint-plugin-import/blob/569d72688fa6ae5c038c51eafa4f6016fc1ee802/utils/resolve.js#L220

notice we are returning a relative path from the resolve method but the variable name refers to an absolute path. 

should we be using `path.resolve` to return an absolute path instead? I am happy to open a PR with the change once confrimed

**Comment by ljharb** [comment]
It's important to use `resolve`, but it's reasonably to pass the result of that into `path.resolve`. A PR would be appreciated.
